components/Cases/Cases.tsx

```diff
@@ -116,7 +87,7 @@ const CasesWrapper = ({ id, person }: WrapperProps): React.ReactElement => {
       ) : (
         <ErrorSummary
```

Comment:
this component has `role='alert'` on it, which isn't correct for this kind of notice. it's intended for errors caused by user action, whereas this is more just informational/notice. [see mdn](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Alert_Role#accessibility_concerns)

Reply:
@jhackett1 Done in https://github.com/LBHackney-IT/lbh-social-care-frontend/pull/362/commits/dc88af1c87f10b87934be72cb625e7a25534c436 and https://github.com/LBHackney-IT/lbh-social-care-frontend/pull/362/commits/08fe07377bd71da55e4ac7f48c96ce9b1b8a87a4

Reply:
Are `roles` solely for accessibility?